package.json

```diff
@@ -45,21 +46,24 @@
     "@types/react": "^16.8.6",
     "@types/react-dom": "^16.8.2",
     "@types/react-redux": "^7.0.8",
+    "@types/react-transition-group": "^4.2.2",
     "@types/remote-redux-devtools": "^0.5.3",
     "dexie": "^2.0.4",
     "express": "^4.16.2",
     "hadouken-js-adapter": "^0.39.1",
     "inversify": "^5.0.1",
+    "markdown-it": "^9.1.0",
     "mkdirp": "^0.5.1",
     "moment": "^2.22.1",
     "node-fetch": "^2.6.0",
     "openfin-service-signal": "^1.0.0",
-    "openfin-service-tooling": "1.0.12",
+    "openfin-service-tooling": "^1.0.14-alpha.56",
     "pre-commit": "^1.2.2",
     "puppeteer": "1.3.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.3",
     "react-redux": "^7.0.2",
+    "react-transition-group": "^4.2.2",
     "redux": "^4.0.1",
     "redux-devtools-cli": "0.0.1-1",
```

Comment:
Side note here - Do we need this, and if we do, can we get it upgraded?  Its flagging multiple high vulnerabilities via `npm audit`

Reply:
It's not coming up for me when I audit, I only get warnings about lodash in the service tooling.

Reply:
Removed in this PR https://github.com/HadoukenIO/notifications-service/pull/122